pkgs/data/fonts/courier-prime/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

Comment:
Can we use only fetchFromGitHub here?

Reply:
I don't understand what you mean.

Reply:
Like https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/amiri/default.nix does it.

Reply:
Ah I see, you mean _not_ using `fetchFromGitHub`. What benefit does that provide? Would the project need to have any releases or tags for that to work? This project [doesn't have either](https://github.com/quoteunquoteapps/CourierPrime/releases).

Reply:
> What benefit does that provide?

It is less complex and if you are just copying files you don't need a full blown stdenv.

> Would the project need to have any releases or tags for that to work?

You are not using one right now, so no.

Reply:
Changed.